data/mods/Aftershock/items/comestibles/alienfood.json

```
@@ -132,5 +173,406 @@
+  },
+  {
+    "id": "alien_liver_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_liver",
+    "looks_like": "meat_cooked",
+    "name": "cooked alien liver",
+    "description": "After some time cooking, the pores of the organ have shrunk, and it's much, much softer - almost yogurt-like in its consistency.  It still smells awful.",
+    "parasites": 0,
+    "price_postapoc": 50,
+    "fun": -2,
+    "healthy": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_brain",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": { "str_sp": "raw alien brains" },
+    "description": "A brain butchered from an alien organism.  It looks somewhat similar to its earthly analogues, barring its light-blue coloration and gland riddled frontal lobe.",
+    "color": "light_gray",
+    "healthy": -15,
+    "price_postapoc": 10,
+    "quench": 10,
+    "fun": -20,
+    "parasites": 15,
+    "calories": 78,
+    "vitamins": [ [ "vitA", 0 ], [ "vitC", 2 ], [ "calcium", 1 ], [ "iron", 8 ], [ "vitB", 54 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_brain_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_brain",
+    "looks_like": "offal_cooked",
+    "name": { "str_sp": "cooked alien brains" },
+    "description": "Fried and scrambled, this doesn't look the most appetizing, but it's edible (you hope).",
+    "price_postapoc": 25,
+    "parasites": 0,
+    "healthy": -2,
+    "fun": -8,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_kidney",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien kidney",
+    "description": "You are guessing here, but this is likely the kidney of the animal.  The ammonia smell was the biggest hint.",
+    "color": "brown",
+    "healthy": -8,
+    "price_postapoc": 25,
+    "quench": -5,
+    "calories": 54,
+    "vitamins": [ [ "vitA", 6 ], [ "vitC", 2 ], [ "calcium", 0 ], [ "iron", 8 ], [ "vitB", 155 ], [ "mutant_toxin", 25 ] ]
+  },
+  {
+    "id": "alien_kidney_cooked",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_kidney",
+    "looks_like": "offal_cooked",
+    "name": "cooked alien kidney",
+    "description": "Thoroughly cooked and prepared, this alien organ has acquired the filthy ammonia smell of a poorly mopped spaceship deck.  Even if it were safe to eat, you are not sure you'd want to.",
+    "healthy": -2,
+    "price_postapoc": 25,
+    "quench": -2,
+    "fun": -3,
+    "parasites": 0,
+    "delete": { "flags": [ "RAW", "PREDATOR_FUN" ] }
+  },
+  {
+    "id": "alien_sweetbread",
+    "type": "COMESTIBLE",
+    "copy-from": "alien_lung",
+    "looks_like": "offal",
+    "name": "raw alien sweetbread",
+    "description": "You got this chunk of meat from the throat of the animal; it looks...  different?",
```

This will need changing to an ellipsis to match the text standard.

---

```suggestion
    "description": "You got this chunk of meat from the throat of the animal; it looks… different?",
```

> The "..." to ellipsis is something I find rather difficult to do for various reasons.

Ideally github action would pick the whole error message up and display it along the offending line, so you can simply copy-paste the fix. However currently only the first line of the error message can be displayed and it also does not display in this PR for some reason.

---

What a mess, thank you Qrox, appreciate it!

---

Sheesh! (Note to self: just use "-" instead of "..." when possible.)